test/actions/admin.test.js

```diff
@@ -871,7 +871,11 @@ describe('Actions.Admin', () => {
             post('/plugins').
             reply(200, testPlugin);
 
-        await Actions.uploadPlugin(testFileData)(store.dispatch, store.getState);
+        await Actions.uploadPlugin(testFileData, false)(store.dispatch, store.getState);
+
+        // TODO: not sure how to add a test for Actions.uploadPlugin(testFileData, true)
```

Comment:
Naively myself, I'd duplicate this test case, call it `uploadPlugin, forced`, and invoke with `, true`, but configuring the `nock` mock to expect `post('/plugins', '?force=true')`.

Reply:
I did now, not sure how meaningful this test is, but probably better than nothing. The flag is passed in the (multipart) form, so the POST URL is the same.